src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -254,7 +255,7 @@ public void onCommandExecuted(CommandExecuted commandExecuted)
 			}
 		}
 		// TODO: Uncomment when the commands will be refactored to com.hydratereminder.command
-		// commandDelegate.invokeCommand(commandExecuted);
+//		 commandDelegate.invokeCommand(commandExecuted);
```

Comment:
This change is not needed in scope of this PR

Reply:
Reverted the change